packages/synapse-react-client/stories/SchemaDrivenAnnotationEditor.stories.tsx

```diff
@@ -5,11 +5,7 @@ import { SchemaDrivenAnnotationEditor } from '../src/lib/containers/entity/annot
 const meta = {
   title: 'Synapse/SchemaDrivenAnnotationEditor',
   component: SchemaDrivenAnnotationEditor,
-  render: args => (
-    <div className="bootstrap-4-backport">
-      <SchemaDrivenAnnotationEditor {...args} />
-    </div>
-  ),
+  render: args => <SchemaDrivenAnnotationEditor {...args} />,
```

Comment:
Remove bootstrap-4-backport class from the story, so that the effect of edits made to the component are clear in the story (e.g. that bootstrap-4-backport styling is required to maintain the current Form formatting)